From a4d556f0357df5a13c13b977a8539d207f7c5119 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Tue, 26 May 2020 21:10:05 -0700
Subject: [PATCH 15/33] Revert "ARM: vexpress: Don't select VEXPRESS_CONFIG"

This reverts commit 848685c25da99d871bbd87369f3c3d6eead661ac.

This causes issues for ARCH=arm allmodconfig and will eventually be
reverted upstream.

Link: https://lore.kernel.org/lkml/CAL_JsqJ9LnCpH92bXfk4ryXrTgzbPM03=PYLTzQXazuznJM=hg@mail.gmail.com/
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 arch/arm/mach-vexpress/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-vexpress/Kconfig b/arch/arm/mach-vexpress/Kconfig
index 8391a5b3cd78..065e12991663 100644
--- a/arch/arm/mach-vexpress/Kconfig
+++ b/arch/arm/mach-vexpress/Kconfig
@@ -19,6 +19,7 @@ menuconfig ARCH_VEXPRESS
 	select POWER_SUPPLY
 	select REGULATOR if MMC_ARMMMCI
 	select REGULATOR_FIXED_VOLTAGE if REGULATOR
+	select VEXPRESS_CONFIG
 	help
 	  This option enables support for systems using Cortex processor based
 	  ARM core and logic (FPGA) tiles on the Versatile Express motherboard,
-- 
2.27.0.rc0

